transport/wseb/src/main/java/org/kaazing/gateway/transport/wseb/WsebSession.java

```diff
@@ -179,6 +179,14 @@ public WsebSession(int ioLayer,
         transportSession.setHandler(transportHandler);
     }
 
+    protected void setIoAlignment0(Thread ioThread, Executor ioExecutor) {
+        transportSession.setIoAlignment(ioThread, ioExecutor);
+        super.setIoAlignment0(ioThread, ioExecutor);
+
+        // TODO shoud we detachWriter() too so that newly attached writer gets aligned by default ?
+        // TODO writer gets aligned now since it is parent. what about reader alignment ?
```

Comment:
Do you have a test that fails without this fix? Writer is okay as you point out since it's the parent. For the reader, I think it might be best to do logic similar to that in WsebSession.attachReader, only without calling attachReader0.